functions/split-user-data.js

```diff
@@ -17,24 +17,70 @@ const subjectColorsCollection = db.collection('user-subject-colors');
 const uniqueIncrementerCollection = db.collection('user-unique-incrementer');
 const onboardingDataCollection = db.collection('user-onboarding-data');
 
-userDataCollection.get().then(userData => {
-  Promise.all(
-    userData.docs.map(userDataDocument => {
-      const batch = db.batch();
+if (process.argv[2] === '--dry-run') {
+  userDataCollection.get().then(userData => {
+    const oldData = [];
+    const newUsernameCollection = {};
+    const newSemestersCollection = {};
+    const newToggleableRequirementChoicesCollection = {};
+    const newSubjectColorsCollection = {};
+    const newUniqueIncrementerCollection = {};
+    const newOnboardingDataCollection = {};
```

Comment:
We could handle this in a different PR if you feel that it's outside of the scope of this PR, but I want to note that we should also make sure to create a collection for `user-selectable-requirement-choices` which is on the dev Firestore but not prod.  This would be reflected in the dry run and the actual data change processes in this script.

Reply:
I don't think it's relevant here. `user-selectable-requirement-choices` is added after we do the split on dev, so it's never been in the user data collection anyways. Prod user also doesn't have this data anyways.
In terms of creating new collections, it's not an issue. Firestore will create a new collection for you automatically whenever you add the first document into that collection.